tests/integration/nautobot-populate.py

```diff
@@ -617,3 +622,16 @@ def make_nautobot_calls(endpoint, payload):
 
 if ERRORS:
     sys.exit("Errors have occurred when creating objects, and should have been printed out. Check previous output.")
+
+
+###############
+# v2.3+ items #
+###############
+if nautobot_version >= version.parse("2.3"):
+    # Create role for virtual machine interfaces
+    logger.debug("Creating VM Interface Role")
+    vm_interface_roles = [
+        {"name": "Test VM Interface Role", "color": "aa1409", "vm_role": False, "content_types": ["virtualization.vminterface"]},
+    ]
+    created_device_roles = make_nautobot_calls(nb.extras.roles, device_roles)
+    logger.debug("VM Interface Role created")
```

Comment:
This needs to be above `if ERRORS:` or else it doesn't stop if there are any errors.